eksconfig/config.go

```diff
@@ -100,6 +100,17 @@ type Config struct {
 	// all resources on creation fail.
 	OnFailureDeleteWaitSeconds uint64 `json:"on-failure-delete-wait-seconds"`
 
+	// InternalKubeControllerManagerQPS is the internal EKS kube-controller-manager qps
+	InternalKubeControllerManagerQPS   string `json:"internal-kube-controller-manager-qps,omitempty"`
+	// InternalKubeControllerManagerBurst is the internal EKS kube-controller-manager burst
+	InternalKubeControllerManagerBurst string `json:"internal-kube-controller-manager-burst,omitempty"`
+	// InternalKubeSchedulerQPS is the internal EKS kube-scheduler qps
+	InternalKubeSchedulerQPS           string `json:"internal-kube-scheduler-qps,omitempty"`
+	// InternalKubeSchedulerBurst is the internal EKS kube-scheduler burst
+	InternalKubeSchedulerBurst         string `json:"internal-kube-scheduler-burst,omitempty"`
+	// InternalFEUpdateMasterFlagsURL is the internal EKS update master flags endpoint
+	InternalFEUpdateMasterFlagsURL     string `json:"internal-fe-update-master-flags-url,omitempty"`
```

Comment:
Move these inside `Parameters`?

Reply:
Make senses. Updated in the next commit.